src/graphql/query/algoliaLoanStatus.graphql

```diff
@@ -8,6 +8,7 @@ query algoliaLoanStatus(
 			values {
 				id
 				status
+				matchingText
```

Comment:
In the future we'll fix up the Algolia backend sync to deal with this but for now we can patch clientside.